bilby/gw/detector/interferometer.py

```diff
@@ -491,6 +491,17 @@ def inject_signal_from_waveform_polarizations(self, parameters, injection_polari
         for key in parameters:
             logger.info('  {} = {}'.format(key, parameters[key]))
 
+    @property
+    def _window_power_correction(self):
+        """
+        This property enables the old (incorrect) PSD correction to be applied
+        using the :code:`BILBY_INCORRECT_PSD_NORMALIZATION` environment variable.
+        """
+        if os.environ.get("BILBY_INCORRECT_PSD_NORMALIZATION", "FALSE").upper() == "TRUE":
```

Comment:
I'd suggest using [`string_to_boolean`](https://github.com/bilby-dev/bilby/blob/main/bilby/core/utils/env.py#L3) here, it means something like `1` would work as well and is consistent with what we do else where.